index.bs

```diff
@@ -1,17 +1,16 @@
 <pre class='metadata'>
 Group: AOM
-Status: FD
-Text Macro: SPECVERSION v1.1.0
+Status: WGD
 Title: AV1 Image File Format (AVIF)
-URL: https://AOMediaCodec.github.io/av1-avif/v1.1.0.html
+URL: https://AOMediaCodec.github.io/av1-avif
 !Latest version: <a href="https://AOMediaCodec.github.io/av1-avif">https://AOMediaCodec.github.io/av1-avif</a>
-!Previously approved version: <a href="v1.0.0.html">v1.0.0</a>
+!Previously approved version: <a href="v1.1.0.html">v1.1.0</a>
 Shortname: av1-avif
 Editor: Cyril Concolato, Netflix, cconcolato@netflix.com
```

Comment:
Move me to the list of former editors and add the new editors.

Reply:
Not sure who to put here so I just added myself for now. Contributors to be added on the next pull requests.